travis.sh

```diff
@@ -51,6 +51,8 @@ if [[ "$ROS_DISTRO" == "kinetic" ]] && ! [ "$IN_DOCKER" ]; then
   travis_time_start build_docker_image
   docker build -t industrial-ci/xenial .ci_config
   travis_time_end  # build_docker_image
+
+  travis_time_start run_travissh_docker
```

Comment:
Nesting works. You can see an example at [a Travis result](https://travis-ci.org/130s/industrial_ci/jobs/132129721#L773) I experimented in my own repo; expand `travis_time_start run_travissh_docker` and you'll see child folders. Level of the layers isn't visualized though.